README.md

```diff
@@ -61,6 +61,22 @@ RuboCop::RakeTask.new do |task|
 end
 ```
 
+## Rails configuration tip
+
+If you are using Rails 6.1 or newer, add the following `config.generators.after_generate` setting to
+your config/application.rb to apply RuboCop autocorrection to code generated by `bin/rails g`.
+
+```ruby
+# config/application.rb
+module YourCoolApp
+  class Application < Rails::Application
+    config.generators.after_generate do |files|
+      system('bundle exec rubocop -A ' + files.join(' '), exception: true)
```

Comment:
Would `-a` have a different result for this?

Reply:
It shows `-A` intentionally. I've added that intent to the documentation.

Reply:
Should this be `files.shelljoin` instead? What if some homebrew generator spits out filenames with spaces or escape sequences?

Passing raw strings to `system` is scary. 😨

Reply:
Good cath! `shelljoin` would be a better choice.